### ISSUE TITLE
Penile swelling

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10015]
**Preferred term label:**
Penile swelling

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Paraphimosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000087
- label: Penile swelling
- parents: [Abnormal penis morphology (HP:0000036)](https://hpo.jax.org/app/browse/term/HP:0000036)
- pmid: PMID:11142469
- def: Abormal enlargement of the penis owing to accumulation of fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10015
- synonyms: Penile edema(NONE)
